src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -1482,7 +1482,8 @@ public async Task AddLayer(Layer layer, bool isBasemapLayer = false)
     {
         if (isBasemapLayer)
         {
-            Map!.Basemap?.Layers.Add(layer);
+            Map!.Basemap ??= new Basemap();
```

Comment:
maybe update the comment to explain that it will create a base map if it doesn't exist?